src/pages/TestDrive/TestDrive.tsx

```diff
@@ -0,0 +1,117 @@
+import { Button, Container, Form, Card } from "react-bootstrap";
+import { useLocation, useParams, Link, useNavigate } from "react-router-dom";
+import React, { useState } from "react";
+import "./TestDrive.scss";
+function TestDrive() {
+  let navigate = useNavigate();
+  const [mobileValid, setMobileValid] = useState(false);
+  const [emailValid,setEmailValid] = useState(false);
+  var pattern = /^(\+\d{1,3}[- ]?)?\d{10}$/;
+  let emailPattern = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/
```

Comment:
these patterns should also be part of the constant so that they can be used across the app.